src/components/TeamMemberTasks/TeamMemberTask.jsx

```diff
@@ -85,6 +85,11 @@ const TeamMemberTask = ({
                             <Link to={task.projectId ? `/wbs/tasks/${task._id}` : '/'}>
                               <span>{`${task.num} ${task.taskName}`} </span>
                             </Link>
+                            <FontAwesomeIcon
+                              className="team-member-tasks-copy"
+                              icon={faCopy}
+                              onClick={() => {navigator.clipboard.writeText(task.taskName)}}
+                            />
```

Comment:
nice work! The only thing I would consider is creating a separate function instead of handling the logic within the anonymous fn. To improve readability but also in case this copy to clip board functionality were to be reused.

Reply:
To take it a step further, if there isn't already a Clipboard component, could turn this into its own component that receives the toast success text and writeText value as props, to make it more reusable.

Reply:
Created a Clipboard component that receives the toast success text and writeText value. Moved the inline logic handling to a separate function inside the Clipboard component.

Reply:
This is great!